leenlab2/reminiscence
# Interface for insert/removing tape to TV

Addresses https://trello.com/c/K7kHAt52/58-interface-for-inserting-a-tape

# Description


- Created Scriptable Object on the Tape so it holds contents of its videos
- Player can now pick up tape and put into TV. TV should correctly play the contents in that tape. Players can remove the tape from the TV and the TV will stop playing that tape. 

### Type of change






New Feature
## How to Integrate

Need to add TapeInformation script to Tape, and attach the tape's SO to the script as a variable. The rest should just be script changes.

# How Has This Been Tested?


- Open Scene TestNewTV. 
- Press T on keyboard and try playing the TV. Nothing should play as it has no inserted tape yet.
- Pick up the tape and test that you can drop it if (1) your crosshair is on nothing else and (2) your crosshair is on another light object
- Pick up the tape and walk to the TV. Press E(if on keyboard) to insert the tape into the TV. It should disappear from your hands. Open the TV again and check that you can play it. The corrupted tape should play.
- Exit the TV and press E on the TV. The tape should reappear in your hands.
- Open the TV again and see if you can play anything. You should be unable to.
- Put the tape back in the TV.
- Complete the puzzle by placing the stool, chair, and lamp in the appropriate spots. Then check the TV to make sure the fixed tape is playing.
        
# Checklists
## Integration Checklist

## Review Checklist
